src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './variables';
+@import './mixin';
+@import './main';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: $main-color;
+}
+
 body {
   margin: 0;
+  padding: 0;
 }
```

Comment:
Consider moving these styles to a separate file to keep the main file clean and clear